administration/src/mui-modules/application/forms/AddressForm.tsx

```diff
@@ -20,14 +22,12 @@ const SubForms = {
 
 type State = CompoundState<typeof SubForms>
 type ValidatedInput = AddressInput
-type Options = {}
-type AdditionalProps = {}
 
-const AddressForm: Form<State, Options, ValidatedInput, AdditionalProps> = {
+const AddressForm: Form<State, ValidatedInput> = {
   initialState: { ...createCompoundInitialState(SubForms), country: { shortText: 'Deutschland' } },
   getArrayBufferKeys: createCompoundGetArrayBufferKeys(SubForms),
   validate: createCompoundValidate(SubForms, {}),
-  Component: ({ state, setState }) => (
+  Component: ({ state, setState }: FormComponentProps<State>) => (
```

Comment:
Can `FormComponentProps<State>` no longer be inferred?

Reply:
It can still be inferred, but we have a linting rule that requires all props of components to be explicitly defined. Personally, I think this rule makes sense to
a) make it obvious which props are needed
b) avoid any excess/wrong props